packages/system-data/src/fields/settings.yaml

```diff
@@ -200,6 +200,60 @@ fields:
       placeholder: $t:unlimited
     width: half
 
+  - field: public_registration_divider
+    interface: presentation-divider
+    options:
+      icon: person_add
+      title: $t:fields.directus_settings.public_registration
+    special:
+      - alias
+      - no-data
+    width: full
+
+  - field: public_registration
+    interface: boolean
+    note: $t:fields.directus_settings.public_registration_note
+    width: half
+    options:
+      label: $t:enabled
+    special:
+      - cast-boolean
+
+  - field: public_registration_role
+    interface: select-dropdown-m2o
+    note: $t:fields.directus_settings.public_registration_role_note
+    options:
+      template: '{{ name }}'
+      filter:
+        admin_access:
+          _eq: false
+    special:
+      - m2o
+    width: half
+    display: related-values
+    display_options:
+      template: '{{ name }}'
+
+  - field: public_registration_verify_email
```

Comment:
Personally I'd swap verify email and the email filter, since the email filter is more closely related to the user role (conceptionally) and the send verification email more related to a process of the registration as a whole.

Reply:
I prefer the toggles being on the left side while the more configurable fields live on the right. Since this is just an opinion thing I'd leave it up to @benhaynes / @rijkvanzanten - For quick reference here is how it currently looks like:

![image](https://github.com/directus/directus/assets/14810858/28e9efef-c9b6-424f-99c9-6f59398a9cf8)